packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -139,7 +139,7 @@ const ripples = {
           delete el.dataset.previousPosition
         }
 
-        animation.parentNode && el.removeChild(animation.parentNode)
+        animation.parentNode && el === animation.parentNode.parentNode && el.removeChild(animation.parentNode)
```

Comment:
I haven't tested this, but could we just use optional chaining?

`if(animation?.parentNode?.parentNode === el) el.removeChild(animation.parentNode)`

Reply:
```suggestion
        if(animation?.parentNode?.parentNode === el) el.removeChild(animation.parentNode)
```